tests/manager/api/test_odl.py

```diff
@@ -55,7 +55,7 @@ def test_get_license_status_document_success(
         # With a new loan.
         loan, _ = odl_api_test_fixture.license.loan_to(odl_api_test_fixture.patron)
         odl_api_test_fixture.api.queue_response(
-            200, content=json.dumps(dict(status="ready"))
+            201, content=json.dumps(dict(status="ready"))
```

Comment:
Yes. We get 200 back on existing loans, and 201 on loans that haven't been fulfilled yet, so I want to check both cases.

Reply:
Further on in this test function there is a test for an existing loan, and its returning 200, so we are effectively testing both now. Before this response was a bit misleading, because it is a new loan response, but we had it mocked as returning 200 instead of 201.